packages/happo-target-react-native/src/defaultOptions.js

```diff
@@ -0,0 +1,96 @@
+const path = require('path');
+
+const iosAppPath = path.join(
+  __dirname, // path of this file
+  '../', // root of this module
+  'runner/ios/build/Build/Products/Debug-iphonesimulator/HappoRunner.app',
+);
+
+const androidAppPath = path.join(
+  __dirname, // path of this file
+  '../', // root of this module
+  'runner/android/app/build/outputs/apk/app-debug.apk',
```

Comment:
Similarly, each of these directories should be separate args for max cross-platform compatibility.